src/utils/utils.tsx

```diff
@@ -0,0 +1,51 @@
+import { WrappedDocument } from "../types";
+import { saveAs } from "file-saver";
+import JSZip from "jszip";
+
+interface generateFileNameI {
```

Comment:
let's avoid `I` suffix if possible

```suggestion
interface GenerateFileName {
```

Reply:
ok got it